react-charts/area-chart/step6.md

```diff
@@ -27,29 +15,40 @@ padding={{
 }}
 </pre>
 
-- This will add more padding to the bottom and left properties to accommodate the axis labels.
+3) <strong>Update the padding for the right value to accommodate the legend.</strong>
 
-4) Locate the code that looks like the following:
+`right: 200`
 
-<pre class="file">
-&lt;ChartGroup&gt;
-...
-&lt;/ChartGroup&gt;
+4) <strong>Add the `legendData` property to the `<Chart>` component.</strong>
+

```

Comment:
The legend is not visible at this point without also setting legendOrientation. By default, the legend is horizontally oriented, so most of it is outside the viewport.

Reply:
For me I see it appear like this, I don't necessarily think its a problem, because it shows the user that its beneficial to them to change the orientation of the legend in the next step. Maybe I could warn them that in the next step they will update the orientation

<img width="646" alt="Screen Shot 2019-12-02 at 1 41 46 PM" src="https://user-images.githubusercontent.com/20118816/69985588-80527d00-1509-11ea-8f26-76df1dc7b656.png">

Reply:
IMO, it's too confusing to explain why this appears broken. Perhaps, we can set the orientation first?

Reply:
Okay updated the order of the steps so that you set the orientation first, thanks for that feedback